src/components/Summary/AlterationSummary.vue

```diff
@@ -140,10 +140,13 @@
           </v-col>
 
           <v-col cols="9" class="inner-col-2">
-            <p class="info-text">Select the date and time of alteration of your business. You may select a date and
-              time up to 10 days in the future (note: there is an <strong>additional fee of $100.00</strong> to enter
-              an alteration date and time in the future). Unless a business has special requirements, most businesses
-              select an immediate Alteration Date and Time.
+            <p
+              id="effective-date-time-instructions"
+              class="info-text">
```

Comment:
These could probably fit on 1 line. We usually (i think*) go to multi line only when it doesn't fit.

Reply:
Yes, or we try to follow the style of the surrounding code, to make the code look nicer and thus be easier to review :)

Reply:
Note that paragraph elements are block-level, so this
```
<p>blah</p>
```
displays the same as
```
<p>
   blah
</p>
```

... Which is to say, follow whatever style looks best in this scenario.

Reply:
ok.